tasks/urls.py

```diff
@@ -5,4 +5,5 @@
     path('', views.list_task, name='tasks'),
     path('login/', views.LoginView.as_view(), name='login'),
     path('register/', views.RegisterView.as_view(), name='register'),
+    path('home/', views.home, name='home'),
```

Comment:
this looks like a website... nice thinking. lets keep this as site url. 

```suggestion
    path('site/', views.home, name='website'),
```